lib/senders/templates/new_device_login.html

```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <title>{{t "Firefox Accounts"}}</title>
+  <title>{{t "Manage account:"}} {{{ link }}}</title>
```

Comment:
@ArkadiyVoronov why are you changing the <title> tag ?

Reply:
I thought you had this line to fix it.